js/navigation/NavigationLayout.ios.js

```diff
@@ -2,22 +2,93 @@
  * @providesModule NavigationLayout
  * @flow
  */
-
 import {
   StackNavigator,
   TabNavigator
 } from 'react-navigation'
-
 import SignInSignUpScreen from '../scenes/SignInSignUpScreen'
 import AccountScreen from '../scenes/AccountScreen'
 import MyPotLucksScreen from '../scenes/MyPotlucksScreen'
-import CreatePotluckScreen from '../scenes/CreatePotluckScreen'
+import InfoScreen from '../scenes/InfoScreen'
+import FoodScreen from '../scenes/FoodScreen'
+import InvitesScreen from '../scenes/InvitesScreen'
+import { colors, typography, windowWidth, windowHeight } from '../styles/baseStyles'
 
+const HostTabNavigator = TabNavigator(
+  {
+    Info: { screen: InfoScreen },
+    Food: { screen: FoodScreen },
+  },
+  {
+    initialRouteName: 'Info',
+    tabBarPosition: 'top',
+    tabBarOptions: {
+      activeTintColor: 'white',
+      inactiveTintColor: colors.mainBrandColor,
+      activeBackgroundColor: colors.mainBrandColor,
+      labelStyle: {
+        fontSize: typography.baseFontSize,
+        justifyContent: 'center',
+        paddingBottom: 3, //best option?
+        
+      },
+      tabStyle: {
+        borderRight: 1,
+      },
+      style: {
+        width: windowWidth*.9,
+        height: windowHeight*.04,
+        marginTop: 10,
+        borderRadius: 5,
+        alignSelf:'center',
+        overflow: 'hidden',
+        borderWidth: 1,
+        borderTopWidth: 1,
+        borderColor: colors.mainBrandColor,
+        borderTopColor: colors.mainBrandColor,
+        
+        }
+      }
+    }
+)
+const GuestTabNavigator = TabNavigator(
+    {
```

Comment:
TabNavigator takes two pretty verbose objects here. Could you put them into named variables so that we know what their purpose is?

Also, fix your indentation.